com.unity.netcode.gameobjects/Editor/HiddenScriptEditor.cs

```diff
@@ -0,0 +1,56 @@
+using Unity.Netcode.Components;
+using Unity.Netcode.Transports.UNET;
+using Unity.Netcode.Transports.UTP;
+using UnityEditor;
+
+namespace Unity.Netcode.Editor
+{
+    public class HiddenScriptEditor : UnityEditor.Editor
```

Comment:
Can you add docs to all of these classes so our "missing xmldoc warnings" list doesn't continue to grow?

Suggestion:
```
/// <summary>
/// Internal use.
/// </summary>
```

Reply:
Sure